src/vellum/workflows/nodes/core/inline_subworkflow_node/node.py

```diff
@@ -24,6 +26,7 @@ class InlineSubworkflowNode(BaseSubworkflowNode[StateType], Generic[StateType, W
     """
 
     subworkflow: Type["BaseWorkflow[WorkflowInputsType, InnerStateType]"]
+    subworkflow_inputs: ClassVar[Union[EntityInputsInterface, BaseInputs]]
```

Comment:
Ideally this is parameterized with `WorkflowInputsType` instead of `BaseInputs`. In order to do so, would require some custom mypy-ing bc they don't have great support for typevar's used in this way

Reply:
https://app.shortcut.com/vellum/story/5986